students/steve_walker/lesson03/list_lab.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+#Series 1
+fruitlist = ["Apples", "Pears", "Oranges", "Peaches"]
+print(fruitlist)
+
+add_fruit = input("\nAnother fruit, perhaps? What would you like?\n")
+fruitlist.append(add_fruit)
+print(fruitlist)
+
+number = int(input("\nPick a number from 1 to " + str(len(fruitlist)) + ".\n"))
+print(number, ": ", fruitlist[number-1],"\n")
+
+fruitlist = ["Guava"] + fruitlist
```

Comment:
You'll want snake_case variables as well as function names in Python.  So, fruit_list.